src/contract.rs

```diff
@@ -55,20 +57,31 @@ pub mod execute {
         deps: DepsMut,
         env: Env,
         deposit_ids: Vec<u32>,
+        profit_taking_or_stop_loss: Vec<bool>,
     ) -> Result<Response<PalomaMsg>, ContractError> {
         let state = STATE.load(deps.storage)?;
+        if deposit_ids.len() != profit_taking_or_stop_loss.len() {
```

Comment:
This condition winds up being a little wonky, so what I think you'd like is:

```
#[cw_serde]
pub struct Deposit {
        id: u32,
        profit_taking: bool,
}

#[cw_serde]
pub enum ExecuteMsg {
    PutWithdraw(Vec<Deposit>),
}
```

Reply:
fixed